vue-frontend/src/components/jobs/PerksAndBenifits.vue

```diff
@@ -201,6 +201,14 @@ export default {
   flex-direction: column;
 }
 
+.flex-div div > img {
```

Comment:
UI is breaking, Fix it

![Screenshot from 2022-03-11 15-54-06](https://user-images.githubusercontent.com/69897605/157849237-03e79628-e718-4b6a-9be0-66d2ff88182e.png)